### DISCUSSION TITLE
VDH asks to download companion app even after I've downloaded/installed it.

### DISCUSSION THREAD

**Discussion by RLynwood** [aclap-dev/video-downloadhelper#381] [Help and Q&A]
### Your question

Using VDH 8.2.2.8 stable in traditional (unsnapped) Firefox 123.0.1, 64-bit, in Kubuntu 23.10, I believe I've downloaded and installed VDH's companion app, but when I attempt to download a video, VDH says I have to install the companion app.  Can someone solve this for me?

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unneeded
platform: x86-64 linux
UA: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: no.

**Comment by Mattmjs** [comment]
https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-not-recognized#linux-all

**Comment by jf-clavel** [comment]
Hello
Windows 64 bits
Je rencontre le même problème depuis le 5/04/2024 , sous firefox version 8.2.2.8 ou sous opera version 8.2.2.9 Suppression, reset,  et réinstallation sont sans effet. De plus impossible de changer le dossier de téléchargement par défaut. La combobox du choix du dossier ne fonctionne pas.

I have encountered the same problem since 04/5/2024, under Firefox version 8.2.2.8 or under Opera version 8.2.2.9 removing the extension, reset, and reinstallation have no effect. Also impossible to change the default download folder. The folder selection combo box does not work.
![anomalie vdh](https://github.com/aclap-dev/video-downloadhelper/assets/62204834/e90b1ac0-9dd7-4814-a1ce-2876dcdc744f)

**Reply by jcfrog** [reply]
bonjour, quelles sont vos versions de coapp?

**Comment by jf-clavel** [comment]
Bonjour

 

windows 64 bits 23H2 build 22631.3296

opera :

GX LVL5 (core: 107.0.5045.86) version de chromium 121.0.6167.186

extension opera Video download helper 8.2.2.9(stable)

 

firefox 124.0.2 

extension firefox Video download helper 8.2.2.8 (stable)

 

appli compagnon : downloadhelper coapp 2.019.0 (32bits) : pourquoi 32 bits ? alors que j’ai téléchargé la version pour win 64.

 

J’ai supprimé la copp avec les utilitaires windows (revouninstaller) , puis réinstallé la dernière version téléchargée 2.019.0. L’appli compagnon est maintenant reconnue sous firefox et opera.

 

Pour le changement du dossier de téléchargement par défaut il y a eu un progrès : 

avec firefox : tout est  ok je peux changer le dossier par défaut

avec opera je peux maintenant  accéder à la liste des disques et des dossiers mais après en avoir choisi un : ‘’Attempting to use a disconnected port object’’

 

 

Cordialement

 

JF Clavel

 

 

 

De : jcfrog ***@***.***> 
Envoyé : dimanche 7 avril 2024 09:28
À : aclap-dev/video-downloadhelper ***@***.***>
Cc : jf-clavel ***@***.***>; Comment ***@***.***>
Objet : Re: [aclap-dev/video-downloadhelper] VDH asks to download companion app even after I've downloaded/installed it. (Discussion #381)

 

bonjour, quelles sont vos versions?
